talk/expert/cpp20concepts.tex

```diff
@@ -0,0 +1,194 @@
+\subsection[tmpl]{C++20 Requirements and Concepts}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{C++20 Requirements and Concepts}
+  \begin{block}{Motivation}
+    Generic programming is made of functions and class templates
```

Comment:
Technically also variable templates.

Reply:
"... is made of variable, function and class templates..." ?